nvme.c

```diff
@@ -3692,7 +3692,7 @@ static int sec_send(int argc, char **argv, struct command *cmd, struct plugin *p
 	}
 
 	err = nvme_sec_send(fd, cfg.namespace_id, cfg.nssf, cfg.spsp, cfg.secp,
-			buf_size, buf_size, sec_buf);
+			cfg.tl, cfg.tl, sec_buf);
```

Comment:
At this point, every caller to this function now has the same value for the 6 and 7 parameters, they should be collapsed into the same one.

Reply:
> At this point, every caller to this function now has the same value for the 6 and 7 parameters, they should be collapsed into the same one.

I added a commit that removed the redundant parameter to nvme_sec_send. However, it raises another issue: the same issue applies to nvme_sec_recv, where "al" and "size" are redundant parameters passed from the user command line. I didn't address that since one of the parameters that affects the command-line user interface needs to be removed, and I thought the maintainers might have some preference about which one to remove/keep.